frontend/packages/console-dynamic-plugin-sdk/src/extensions/notification-alert.ts

```diff
@@ -8,7 +9,7 @@ export type AlertAction = ExtensionDeclaration<
     /* Action text */
     text: string;
     /* Function to perform side effect */
-    action: CodeRef<(alert) => void>;
+    action: CodeRef<(alert, launchModal?: LaunchModal) => void>;
```

Comment:
```suggestion
    action: CodeRef<(alert: Alert, launchModal: LaunchModal) => void>;
```

Why optional ?
I think we can also add type for `alert` arg

Reply:
Not everyone would require modal launching capabilities.

Reply:
true, in that case the consumer can still pass just a function which looks at first param

making the `launchModal` optional makes it seem like `console` will not always pass `launchModal` arg which is not true

Reply:
makes sense!